pkg/landscaper/installations/executions/template/gotemplate.go

```diff
@@ -480,3 +483,62 @@ func resolveComponents(defaultCD *cdv2.ComponentDescriptor, list *cdv2.Component
 
 	return components, nil
 }
+
+func getImageVectorGoFunc(cd *cdv2.ComponentDescriptor, list *cdv2.ComponentDescriptorList) func(args ...interface{}) map[string]interface{} {
```

Comment:
```suggestion
func generateImageVectorGoFunc(cd *cdv2.ComponentDescriptor, list *cdv2.ComponentDescriptorList) func(args ...interface{}) map[string]interface{} {
```
to keep the same naming in spiff and go

Reply:
This is done.